service-base/src/main/java/org/eclipse/hono/service/registration/AbstractRegistrationService.java

```diff
@@ -314,18 +398,21 @@ protected CacheDirective getRegistrationAssertionCacheDirective(final String dev
     }
 
     /**
+     * @deprecated As of release 1.2.0, replaced by {@link #getSupportedGatewaysForDevice(String, String, JsonObject, Handler)}
```

Comment:
Considering the above mentioned community call decision, could we remove this deprecated method too since it is not part of the publicly documented API? To me it looks this is the case but I might forget something.

Reply:
I did not see your comment below earlier, where already mentioned as well that the old version of `getSupportedGatewaysForDevice` can be removed. So I will remove it.